migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.adoc

```diff
@@ -58,6 +51,17 @@ If you are not using gateways, and have verified your cluster-wide migration, cr
 Before creating a default revision tag and relabelling the namespaces, you must migrate all remaining workload namespaces, including gateways.
 ====
 
-// insert xref to "Creating the default revision tag and relabeling the namespaces" after PR OSSM-9000 is merged
+include::modules/ossm-creating-a-default-revision-tag-and-relabeling-the-namespaces.adoc[leveloffset=+2]
+
+include::modules/ossm-migrating-a-cluster-wide-deployment-using-the-istio-injection-label-with-cert-manager.adoc[leveloffset=+1]
+
+include::modules/ossm-migrating-workloads-using-the-istio-injection-label-with-cert-manager.adoc[leveloffset=+2]
+
+.Next steps
```

Comment:
Next steps should be also the same as in `Migrating workloads by using the istio injection label`
And we will have to add the `Creating the default revision tag and relabeling the namespaces` chapter or just point users to that chapter.
`Migrating a cluster-wide deployment by using the istio injection label` must have 3 parts: migrating deployment + migration workloads + creating default tag.

Reply:
done